### ISSUE TITLE
How can I adapt this code for Arduino?

### ISSUE THREAD

**Issue by Dancopy** [WiseLord/hub08lib#1]
Greetings friend! Do you have this same code for Arduino?
Or how can I adapt it to Arduino? Thank you

**Comment by WiseLord** [comment]
No, I don't like and don't know Arduino well, so I have no idea how you
could do it.
Why don't you want to use this pure C project? It's much faster and gives
best refresh rate than arduino C++ code could provide.

сб, 20 окт. 2018 г., 21:33 Dancopy <notifications@github.com>:

**Comment by Dancopy** [comment]
First, thanks for the reply!
Is that I do not know how to write this code in the microcontroller; Can you help me or clarify how can I do it? Which program should I use?  Thanks++

**Comment by WiseLord** [comment]
> First, thanks for the reply!
> Is that I do not know how to write this code in the microcontroller; Can you
> help me or clarify how can I do it? Which program should I use?  Thanks++

You should flash it via ISP interface of ATmega. I use USBASP device for it.
Among software you can use (as me), avrdude from terminal. Also there is a big 
amount of flashers with GUI availiable.

It should be easy to google how to flash AVR microcontrollers.

**Comment by Dancopy** [comment]
Ok, and in that list of files, in your code, which file should be used (open in the program) to write to the ATmega microcontroller? Thanks

**Comment by WiseLord** [comment]
None. This is the source code. You should compile it and you'll find hex
file to be flashed in the "build" directory.

вс, 21 окт. 2018 г., 23:45 Dancopy <notifications@github.com>:

**Comment by Dancopy** [comment]
Okay, but what's the easiest way to write these files to an atmega328p Microcontroller? What steps, therefore, I have no experience with; if you can help me, I thank you very much!

PS. From this bunch of your files, which one should I use to write to the Microcontroller?

**Comment by Dancopy** [comment]
stevanov put this file here (https://github.com/stevanov/hub08) to work with arduino but it has some errors; Is it your knowledge?
I would very much like to make this watch with this led module;
Thank you+

**Comment by Dancopy** [comment]
Any updates to this code for Arduino?
This file: https://github.com/stevanov/hub08 by stevanov is giving error in compiling Arduino IDE.
Thanks

**Comment by WiseLord** [comment]
В письме от вторник, 23 июля 2019 г. 01:24:21 +03 пользователь Dancopy 
написал:
> Any updates to this code for Arduino?
> This file: https://github.com/stevanov/hub08 by stevanov is giving error in
> compiling Arduino IDE. Thanks

This code is not for Arduino. It's only for ATmega328p. If you're experienced 
in C/C++ you can make the port, of course. But it is expected to be much 
slower when using Arduino libraries instead of pure work with ports.
